sample/src/main/java/com/immutable/sdkdemo/SdkDemo.kt

```diff
@@ -1,7 +1,5 @@
 package com.immutable.sdkdemo
 
-import com.immutable.sdk.api.CollectionsApi
-
 fun main() {
-    println("List of collections: ${CollectionsApi().listCollections().result.joinToString { it.name }}")
+    // Todo: print list of collections
```

Comment:
Is this on purpose?

Reply:
Yeah will re-add once the collections api is added. Can do that one next.